src/input/mod.rs

```diff
@@ -2176,6 +2201,7 @@ fn should_intercept_key(
     mods: ModifiersState,
     screenshot_ui: &ScreenshotUi,
     disable_power_key_handling: bool,
+    is_inhibiting_shortcuts: bool,
```

Comment:
I wonder, do other compositors have any hard-coded key to escape the input inhibition?

Reply:
Not necessarily that i know of, but [the protocol description](https://wayland.app/protocols/keyboard-shortcuts-inhibit-unstable-v1#zwp_keyboard_shortcuts_inhibitor_v1) mentions that it is fully compliant for the compositor to keep such a keyboard shortcut active even when other shortcuts are generally inhibited. I'm imagining this shouldn't be hardcoded, but rather configurable. Like any other bind. I feel i'd want it on my system to be something like `Mod+Ctrl+Shift+Escape` (and maybe just `Super+Escape` for the application-specific release. e.g. lan-mouse has its own built-in release that doesn't need compositor support, which is very useful already, and for such well-behaved clients i don't even need a compositor release)

Reply:
It's configurable in mutter it seems